src/app/app.module.ts

```diff
@@ -1,9 +1,15 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
 import { HttpClientModule } from '@angular/common/http';
 
 import { NgReduxModule, NgRedux, DevToolsExtension } from '@angular-redux/store';
+import {
```

Comment:
Consider separating your redux registration into it's own feature module (which is only imported a single time in the appmodule).

Reply:
Will Do.